addon/components/help-widget/keyboard-shortcuts/combo.hbs

```diff
@@ -0,0 +1,4 @@
+<span class="ember-help-widget-keyboard-combo">
+  {{#each @combo as |key|}}<kbd>{{key}}</kbd>{{/each}}
```

Comment:
Somehow I've never heard of the `<kbd>` before. Had to look it up.

Reply:
We use it in the Help Widget in Members. I wouldn't have known about it otherwise. 😆